pyro/distributions/torch.py

```diff
@@ -125,12 +125,25 @@ def expand(self, batch_shape):
 
 
 class Independent(torch.distributions.Independent, TorchDistributionMixin):
+    @constraints.dependent_property
+    def support(self):
+        return IndependentConstraint(self.base_dist.support, self.reinterpreted_batch_ndims)
+
+    @property
+    def _validate_args(self):
+        return self.base_dist._validate_args
+
+    @_validate_args.setter
+    def _validate_args(self, value):
+        self.base_dist._validate_args = value
+
     def expand(self, batch_shape):
         batch_shape = torch.Size(batch_shape)
         validate_args = self.__dict__.get('_validate_args')
```

Comment:
Hmm I think we actually don't want to pass through `_validate_args`, since we're treating this as a special property with a setter. I think we should instead 
```py
return type(self)(base_dist, self.reinterpreted_batch_ndims)  # <-- no validate_args
```